test.js

```diff
@@ -31,6 +33,13 @@ test('parse-date', (t) => {
 	t.equal(parseDate('20190303', 'Asia/Bangkok'), 1551546000)
 })
 
+test('format-date', (t) => {
+	t.plan(3)
```

Comment:
I don't really mind, but it usually makes sense to use `t.end()` instead to keep the diff smaller in case someone wants to add an additional check.

Reply:
I think it's less safe though. If I want to add an asynchronous assertion, but call `t.end` too early, I don't notice that I intended to have one assertion more. Specifying this explicitly using `t.plan` seems safer.

Reply:
Then it would make sense to be consistent with that in the other tests in the file